spec/models/miq_product_feature_spec.rb

```diff
@@ -112,20 +112,18 @@ def traverse_product_features(product_feature, &block)
     it "returns only visible features" do
       hierarchical_features
       expect(MiqProductFeature).not_to receive(:sort_children)
-      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(
-        %w(widget_copy widget_edit))
+      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(%w(widget_copy widget_edit))
```

Comment:
ugh, "the @Fryguy let me make a cosmetic change with logical changes" strikes back :trollface: 

I keep looking for a change here other than whitespace. SMH.